src/pkgTypes.ts

```diff
@@ -0,0 +1,97 @@
+import type { RawPkg, StyleType } from './@types/pkg';
```

Comment:
Can it be moved to `jsDelivr/`? as it is strictly depend on the fileList provided by the service